<launch>
  
  <group> 
  name='turtles'
  <remap from="tf" to="tf_old"/>
  <remap from="base_footprint/cmd_vel" to="cmd_vel"/>
  <param name="background_b" value="10" type="int" />
  <node  name="turtlesim_extended" pkg="turtlesim" type="turtlesim_node" output="screen" respawn="false" />
  <node pkg="rosservice" type="rosservice" name="clear_bg" args="call --wait /kill 'turtle1'" /> 
  <node pkg="rosservice" type="rosservice" name="spawn1" args="call --wait /spawn 0.5 0.5 0.0 'base_footprint'" /> 
  <node pkg="turtle_tf" type="turtle_tf_broadcaster" name="Rover_TF" args= "base_footprint" />
  <node pkg="sr_ros_navi_tutorial" type="setup_field.py" name="test" />
  </group>



  <node pkg="tf" type="tf_remap" name="tf_remapper" output="screen">
   <rosparam param="mappings"> [{old: world, new: map}] </rosparam>
  </node>
  


  
  
</launch>